MeiliSearchBox.vue

```diff
@@ -72,9 +75,20 @@ export default {
     },
 
     update(options) {
+      this.updateTheme(options)
       this.$el.innerHTML =
         '<input id="meilisearch-search-input" class="search-query">'
       this.initialize(options)
+    },
+
+    updateTheme(options) {
+      if (options.enableDarkMode === true) {
+        this.theme = 'dark'
+      } else if (options.enableDarkMode === false) {
+        this.theme = 'light'
+      } else {
+        this.theme = 'auto'
+      }
```

Comment:
This is always going to be true. 
I tried `auto` and a light theme on my computer and it is still `dark` mode

Reply:
Mmh, with the same computer but on chrome. Let me try again on firefox

Reply:
My bad! I forgot to change the value of enableDarkMode 🤦

Reply:
enableDarkMode is already 'auto' in the playground
https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/5bf8db5acfd158926b187abd6733c4788c372925/playground/.vuepress/config.js#L21

Take your time, no rush 😉